applications/infinity-wallet.md

```diff
@@ -0,0 +1,121 @@
+**![mobile wallet|690x360](https://i.imgur.com/C330Hn2.jpeg)**
+
+
+### Ecosystem Fit
+Wallets are the core component required for any chain, enabling users to interact and onboard to Vechain, and access the Vechain ecosystem (Payments, DApps, DeFi, Web3, NFTs, Decentralized Identity and much more). Making wallets the defining factor for usage and adoption of a chain and its entire ecosystem, however as of today wallets are one of the least well developed applications in the crypto ecosystem.
+
+Today, wallets act mainly as an interface, serving primarily as a transaction signing layer, with very limited functionality or in-wallet features, requiring use of multiple platforms, and are generally lacking in basic UX, while providing very unprofessional, hard to use and aesthetically poor user interfaces, with most platforms built with a focus on developers rather than end users. This has been sufficient for the early adopters in crypto, however, for the next billion users to onboard, a more curated, frictionless and high quality user experience is required!
+
+As of today the Vechain ecosystem has very few wallet solutions and no desktop wallet one-stop super app! The wallet solutions that do exist today are not really built for mass adoption or use by less tech savvy users, they are more built with a focus on usage by the existing tech-savvy user base and generally provide low quality user interfaces, lack a curated in-wallet experience, lack many features/services that users require and overall provide a terrible UX.
+
+Infinity Wallet steps in to fill this need, as a one-stop hub for all payment, DeFi, NFT and Web3 activity. Providing a unique all-in-one wallet and first of its kind Web3 browser solution with a full spectrum of in-platform services, features and protocols, focusing on delivering a premium platform and enhancing the user experience, while providing a user centric interface for a seamless decentralized user experience and new wallet options. Streamlining accessibility and lowering the barrier to entry, to allow for easy onboarding and adoption of DeFi, crypto payments & Web3.
+
+
+## Team 
+Our team is made up from individuals that have extensive experience in the space, with a complete crypto experience from Smart Contracts, DeFi/Web3 projects, DApp development, Wallet and more. Our team has a general background in software development and UI/UX.
+
+Additionally, our team members have between 5-10 years of experience within the space (as investors, miners and developers), so we have a vast amount of experience and understanding of major shortcomings in the market, along with what is needed to enable mass adoption.
+
+Information on the core team members can be found at the following link: https://docs.google.com/document/d/1c4fpyzI64569uNr8l9Ygo_G3x2NTs_Dq8wRGSnOXH14/edit?usp=sharing 
+
+
+## Development Roadmap 
+
+#### Overview
+
+|  | Milestone 1 | Total |
```

Comment:
https://github.com/vechain/grant-program/blob/master/applications/application-template.md#overview-1 We have a rule that limits milestone 1 funds should not be higher than 40% of the total amount you are applying for, I would suggest adding more milestones, eg. Deliverable.1 and Deliverable.2 as milestone 1.

Reply:
Hello @libotony, we would only be requesting grant payment on completion of the grant. Meaning we would cover all upfront costs, deliver all as proposed and only then the grant would be paid, which is why we have a single milestone. This expedites the process and allows us to focus on full completion before receiving any payment, without waiting between milestone payments to proceed to the next milestone.

We can split the milestone up into 2 if you prefer, although we would be submitting both milestones at the same time as we work on full completion.

Reply:
@InfinityWallet We have a rule that defines the first payment will be immediately disbursed after the grant is accepted, could you split the milestone into 2 so that we can proceed internally, we are ok if you submit both milestones.

Reply:
@libotony we have updated the proposal, splitting the deliverables between 2 milestones. If anything else is needed just let us know. Thanks

Reply:
Thanks